lib/matplotlib/axes/_axes.py

```diff
@@ -5300,10 +5300,16 @@ def imshow(self, X, cmap=None, norm=None, aspect=None,
         Display data as an image; i.e. on a 2D regular raster.
 
         The input may either be actual RGB(A) data, or 2D scalar data, which
-        will be rendered as a pseudocolor image. Note: For actually displaying
-        a grayscale image set up the color mapping using the parameters
+        will be rendered as a pseudocolor image. For displaying a grayscale
+        image set up the color mapping using the parameters
         ``cmap='gray', vmin=0, vmax=255``.
 
+        Typically, the number of pixels used for drawing the figure, does not
+        match the number of pixels given in *X*. *interpolation* is used to
+        calculate the displayed pixels, which is inherently prone to visual
+        artifacts. Depending on the ratio of given to need pixels different
+        interpolation methods may be preferable.
```

Comment:
Maybe:  

```python
"""
The number of pixels used to render an image is set by the axes 
size and the *dpi* of the figure, so the image size will usually not 
match the size of *X*, which can lead to aliasing artifacts when the 
image is resampled 
(see :doc:`/gallery/images_contours_and_fields/image_antialiasing`).  
The resampling can be controlled via the *interpolation* kwarg 
and/or :rc:`image.interpolation`, which defaults to 'antialiased'.
"""
```

Reply:
Sorry my suggedtions crosses @story645 please meld as you see fit.

Reply:
I think yours is mostly clearer, but I'd break it up a little:
```The number of pixels used to render an image is set by the axes 
size and the *dpi* of the figure. This can lead to aliasing artifacts when the 
image is resampled because the displayed image size will usually not 
match the size of *X*.```

Reply:
.. agreed, that was a run-on sentence.